src/protocols/at-ps2/mouse_interface.c

```
@@ -367,32 +387,47 @@ void mouse_event_processor(uint8_t data_byte) {
  * function.
  */
 void mouse_input_event_handler() {
```

Cppcheck (2.19.0)

[style] The function 'mouse_input_event_handler' should have static linkage since it is not used outside of its translation unit.

(staticFunction)